hack/run-e2e.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+GOPATH="${GOPATH:-$HOME/go}"
+
+echo "Checking cluster"
+kubectl config current-context
+JSONPATH='{range .items[*]}{@.metadata.name}{"\t"}Ready={@.status.conditions[?(@.type=="Ready")].status}{"\n"}{end}'
+if kubectl get nodes -o jsonpath="$JSONPATH" | grep "Ready=False"; then 
+    exit 1
```

Comment:
Leave a message for the user to set up cluster config properly letting them know it is in ./kube/config or can use custom  EXPORT KUBECONFIG=path